src/Tribe/Views/V2/View.php

```diff
@@ -1708,62 +1719,72 @@ protected function setup_template_vars() {
 		/** @var Rest_Endpoint $endpoint */
 		$endpoint = tribe( Rest_Endpoint::class );
 
+		$category_colors_priority_category = ( tribe_get_event() instanceof WP_Post )
+			? tribe( Category_Color_Priority_Category_Provider::class )->get_highest_priority_category( tribe_get_event() )
+			: [];
```

Comment:
```suggestion
		$category_colors_priority_category = ( tribe_get_event() instanceof WP_Post )
			? tribe( Category_Color_Priority_Category_Provider::class )->get_highest_priority_category( tribe_get_event() )
			: [];
```
Let's avoid calling `tribe_get_event` twice, just store in a var and use that.